iamproperty/iamproperty.github.io
# Ds 60/featured cards

## Summary of Changes:
1. Add a new card deck class variant
2. Add a new featured insights example

## Rationale
Highlight and make the card deck a 'feature' of the page.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/card-deck and /examples/insight-cards

## Ticket(s)
DS-60

## Checklist

## Notes
